services/madoc-ts/src/frontend/site/features/GlobalMenuStack.tsx

```diff
@@ -89,6 +93,12 @@ blockEditorFor(GlobalMenuStack, {
       fullWidth: false,
     },
     maxWidth: null,
+    slug1: '',
+    text1: '',
+    slug2: '',
+    text2: '',
+    slug3: '',
+    text3: '',
```

Comment:
@Heather0K this might be a good place to test a simple case of the `allowMultiple` and I can jump in to fix the capture model short hand. It should allow users to make as many links as they need.